lib/client/bst-config.ts

```diff
@@ -19,8 +19,12 @@ export class BSTConfig {
      * Loads the configuration
      * Done all synchronously as this is done first thing at startup and everything waits on it
      */
-    public static load(): BSTConfig {
-        BSTConfig.bootstrapIfNeeded();
+    public static async load(): Promise<BSTConfig> {
+        try {
+            await BSTConfig.bootstrapIfNeeded();
+        } catch (error) {
+            throw error;
```

Comment:
According to a conversation with @jkelvie will put the logging information errors in a catch in the promise used on the bin files.